src/NosCore.GameObject/ComponentEntities/Extensions/AliveEntityExtension.cs

```diff
@@ -226,7 +226,7 @@ public static CModePacket GenerateCMode(this IAliveEntity aliveEntity)
                 MorphUpgrade = aliveEntity.MorphUpgrade,
                 MorphDesign = aliveEntity.MorphDesign,
                 MorphBonus = aliveEntity.MorphBonus,
-                Size = (aliveEntity as ICharacterEntity)?.Size
+                Size = (aliveEntity as ICharacterEntity).Size
```

Comment:
No need to make it nullable

Reply:
Yes you need it as cmode if mapmonster will crash

Reply:
How can it be a MapMonster if there is a check to see if it is an ICharacterEntity? And overall MapMonsters do have size -> ![img](https://gyazo.com/ce2e91fa0d24b21d8e9c369aaaf0a297)

Reply:
This is not a check this is a cast if this is monster it return null and null doesn’t have a size

Reply:
Mmm... I get it